src/components/Scrollable/Scrollable.jsx

```diff
@@ -107,32 +109,41 @@ export default class Scrollable extends React.PureComponent {
 
     updateScrollbars = () => {
         this.event.next = this.getEvent();
-        // This check ensures that updates (which are a potentially expensive operation)
+
+        const nextEvent = this.event.next,
+            prevEvent = this.event.prev,
+            changed = nextEvent.scrollHeight !== prevEvent.scrollHeight ||
+            nextEvent.scrollWidth !== prevEvent.scrollWidth ||
+            nextEvent.top !== prevEvent.top ||
+            nextEvent.left !== prevEvent.left;
+
+        // Ensures that updates (which are a potentially expensive operation)
         // are only executed if the applicable scroll properties have changed
-        if (!isEqual(this.event.prev, this.event.next)) {
```

Comment:
`isEqual` is not ideal for performance when hammered with scroll events. best to manually check for changes, keeping the check as low-level as possible